app/routes/$.tsx

```diff
@@ -53,18 +52,7 @@ export const loader = async ({ request, params }: Route.LoaderArgs) => {
   handleRedirects(new URL(request.url).pathname);
   handleStaticFileRequests(params["*"]);
 
-  try {
-    let ref = "main";
-    let lang = "en";
-    let doc = await getRepoDoc(ref, `docs/${params["*"]}`);
-    if (!doc) throw null;
-    // FIXME: This results in two fetches, as the loader for the docs page will
-    // repeat the request cycle. This isn't a problem if the doc is in the LRU
-    // cache but we should probably fix it anyway.
-    return redirect(`/docs/${lang}/${ref}/${params["*"]}`);
-    // eslint-disable-next-line no-empty -- should probably do something here
-  } catch {}
-  throw data({}, { status: 404 });
```

Comment:
Shouldn't need any special docs handling in here. Middleware handles most of the cases, and these extra cases we're just not gonna handle